src/vos/vos_ts.h

```diff
@@ -62,6 +62,13 @@ struct vos_ts_pair {
 	struct dtx_id	tp_tx_rh;
 };
 
+struct vos_wts_cache {
+	/** write timestamps */
+	daos_epoch_t	wc_ts_w[2];
```

Comment:
Style: Some comments defining what do "min" and "max" do would be helpful.

Reply:
"min" actually means "second highest", not "minimum"?

Reply:
correct, I changed the names.   Hopefully it's more clear

Reply:
They are much easier to understand now. Thanks.